tests/uki_test.go

```diff
@@ -51,253 +32,304 @@ var _ = Describe("kairos UKI test", Label("uki"), Ordered, func() {
 			fmt.Println(string(serial))
 		}
 		if CurrentSpecReport().Failed() {
-			gatherLogs(vm)
+			//gatherLogs(vm)
```

Comment:
?

Reply:
wooops, good catch! running it locally so I didnt want that much output on failure! Will fix it and merge!